src/main/java/uk/gov/hmcts/reform/roleassignment/util/ValidationUtil.java

```diff
@@ -137,7 +139,6 @@ public static void validateRoleRequest(Request roleRequest) {
             )) {
             throw new BadRequestException(V1.Error.BAD_REQUEST_MISSING_PARAMETERS);
         }
-        validateId(Constants.NUMBER_TEXT_HYPHEN_PATTERN, roleRequest.getAssignerId());
```

Comment:
Any reasons why have we removed this check?

Reply:
Fixed